src/actions/general.js

```diff
@@ -40,12 +42,33 @@ export function resetPing() {
 }
 
 export function getClientConfig() {
-    return bindClientFunc(
-        Client4.getClientConfigOld,
-        GeneralTypes.CLIENT_CONFIG_REQUEST,
-        [GeneralTypes.CLIENT_CONFIG_RECEIVED, GeneralTypes.CLIENT_CONFIG_SUCCESS],
-        GeneralTypes.CLIENT_CONFIG_FAILURE
-    );
+    return async (dispatch, getState) => {
+        dispatch({type: GeneralTypes.CLIENT_CONFIG_REQUEST}, getState);
+
+        let data;
+        try {
+            data = await Client4.getClientConfigOld();
+        } catch (error) {
+            forceLogoutIfNecessary(error, dispatch);
+            dispatch(batchActions([
+                {
+                    type: GeneralTypes.CLIENT_CONFIG_FAILURE,
+                    error
+                },
+                logError(error)(dispatch)
+            ]), getState);
+            return null;
+        }
+
+        Client4.setEnableLogging(data.EnableDeveloper === 'true');
```

Comment:
This is for logging client errors to the server. APIv4 only allows non-admin users to log if developer mode is enabled